query-connector/src/app/shared/page-routes.ts

```diff
@@ -1,44 +1,32 @@
-import { RoleTypeValues } from "../models/entities/user-management";
+import { UserRole } from "../models/entities/user-management";
 
 /**
  * Pathnames
  */
 export enum PAGES {
   LANDING = "/",
   QUERY = "/query",
-  MY_QUERIES = "/queryBuilding",
+  QUERY_BUILDING = "/queryBuilding",
   FHIR_SERVERS = "/fhirServers",
   USER_MANAGEMENT = "/userManagement",
-  GROUP_MANAGEMENT = "/userManagement/userGroups",
```

Comment:
reworked this since Admins did not have access to the base /`userManagement` path, only `/userGroups`. Table content is now controlled at the component level based on the active TableTab